installer/pkg/components/server/configmap.go

```diff
@@ -50,7 +50,7 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 		WorkspaceGarbageCollection: WorkspaceGarbageCollection{
 			ChunkLimit:                 1000,
 			ContentChunkLimit:          1000,
-			ContentRetentionPeriodDays: 21,
```

Comment:
why do we need to change this ? Yaml should override ?

Reply:
If the yaml will override I will change it back.

Reply:
Done.